### PR TITLE
French translation (Continuation)

### PR DESCRIPTION
**Pull request by iNSiGhT59**
French translation :
from line 4636 to 4726
from line 4740 to 4746
from line 4754 to 4787
(line numbers from english string.xml)

### CODE REVIEW

File: res/values-fr/strings.xml [hunk 1]
```diff
@@ -1835,4 +1835,144 @@
...
+    <string name="pie_gap_huge">Enorme</string>
+
+    <string name="pie_center_title">Alignement au milieu</string>
+    <string name="pie_center_summary">Centre la Pie quelque soit l\'endroit où elle est déclenchée</string>
+
+    <string name="pie_stick_title">Eviter la rotation</string>
+    <string name="pie_stick_summary">La Pie reste sur le bord choisi en cas de rotation de l'écran</string>
+
+    <string name="pie_menu_title">Bouton Menu persistant</string>
+    <string name="pie_menu_summary">Toujours afficher le bouton Menu, même quand il n'est pas utile</string>
```
Comment (SferaDev) [inline res/values-fr/strings.xml:1918]: Needs a Backslash on n'est.

File: res/values-fr/strings.xml [hunk 2]
```diff
@@ -1835,4 +1835,144 @@
...
+    <string name="pie_gap_none">Aucun</string>
+    <string name="pie_gap_small">Petit</string>
+    <string name="pie_gap_big">Grand (défaut)</string>
+    <string name="pie_gap_huge">Enorme</string>
+
+    <string name="pie_center_title">Alignement au milieu</string>
+    <string name="pie_center_summary">Centre la Pie quelque soit l\'endroit où elle est déclenchée</string>
+
+    <string name="pie_stick_title">Eviter la rotation</string>
+    <string name="pie_stick_summary">La Pie reste sur le bord choisi en cas de rotation de l'écran</string>
```
Comment (SferaDev) [inline res/values-fr/strings.xml:1915]: Needs a Backslash on l'écran.

File: res/values-fr/strings.xml [hunk 3]
```diff
@@ -1835,4 +1835,144 @@
...
+    <string name="cm_sound_mode_silent">Silencieux</string>
+    <string name="cm_sound_mode_vibrate">Vibreur</string>
+    <string name="cm_sound_mode_sound">Son</string>
+    <string name="cm_sound_mode_soundVibrate">Son &amp; Vibreur</string>
+
+    <!-- Pie controls -->
+    <string name="pie_controls_title">Contrôles de la Pie</string>
+    
+    <string name="pie_gravity_title">Gravité de la Pie</string>
+    <string name="pie_gravity_summary">Choisir l'endroit où sera placée la Pie</string>
```
Comment (SferaDev) [inline res/values-fr/strings.xml:1875]: Needs a Backslash on l'endroit.

File: res/values-fr/strings.xml (hunk 1 above)
Comment (iNSiGhT59) [inline res/values-fr/strings.xml:1918]: How can I edit the xml?

Comment (iNSiGhT59) [inline res/values-fr/strings.xml:1918]: I just merged my two translations in a new one (with added backslash)

Thanks.

**Pull request closed without merging**